packages/odyssey-react/src/components/Toast/Toast.theme.ts

```diff
@@ -13,42 +13,58 @@
 
 export const theme: ThemeReducer = (theme) => ({
-  InAnimationDelay: "0s",
-  OutAnimationDelay: "5300ms",
-  InAnimationDuration: "300ms",
-  OutAnimationDuration: "1000ms",
-  BackgroundColor: theme.ColorBackgroundPrimaryDark,
+  BackgroundColor: theme.ColorBackgroundPrimaryLight,
   BodyFontSize: theme.FontSizeBody,
+  BorderColor: theme.ColorBackgroundPrimaryBase,
   BorderRadius: theme.BorderRadiusBase,
+  BorderStyle: theme.BorderStyleBase,
+  BorderWidth: theme.BorderWidthBase,
   BoxShadow: theme.ShadowHover,
-  CautionBackgroundColor: theme.ColorBackgroundCautionDark,
-  CautionHeadingColor: theme.ColorTextBody,
-  CautionIconColor: theme.ColorTextBody,
-  CautionTextColor: theme.ColorTextBody,
   ColumnGap: theme.SpaceScale1,
-  DangerBackgroundColor: theme.ColorBackgroundDangerDark,
+  MaxWidth: theme.FontLineLengthMax,
+  PaddingBlock: theme.SpaceScale3,
+  PaddingInline: theme.SpaceScale3,
+  RowGap: 0,
+  TextColor: theme.ColorTextBody,
+
+  DismissInsetBlockStart: theme.SpaceScale3,
+  DismissInsetInlineEnd: theme.SpaceScale3,
   DismissedAnimationDelay: "1000ms",
   DismissedAnimationDuration: "1000ms",
-  DismissInsetBlockStart: theme.SpaceScale0,
-  DismissInsetInlineEnd: theme.SpaceScale0,
+
   HeadingFontSize: theme.FontSizeHeading6,
   HeadingFontWeight: theme.FontWeightBold,
   HeadingLineHeight: theme.FontLineHeightHeading6,
-  HeadingMarginBlock: 0,
+  HeadingMarginBlock: theme.SpaceScale0,
   HeadingMarginInline: 0,
-  HeadingPaddingInlineEnd: theme.SpaceScale3,
-  HeadingTextColor: theme.ColorTextBodyInverse,
-  HoverOutAnimationDelay: "300s",
+  HeadingTextColor: theme.ColorTextBody,
+
+  IconColor: theme.ColorPrimaryBase,
+  IconInsetBlockStart: theme.SpaceScale3,
+  IconInsetInlineStart: theme.SpaceScale3,
+  IconMargin: theme.SpaceScale3,
   IconSize: theme.FontSizeHeading4,
-  IconColor: theme.ColorTextBodyInverse,
-  MaxWidth: theme.FontLineLengthMax,
-  PaddingBlock: theme.SpaceScale3,
-  PaddingInline: theme.SpaceScale1,
+
+  InAnimationDelay: "0s",
+  InAnimationDuration: "300ms",
+  HoverOutAnimationDelay: "300s",
+  OutAnimationDelay: "5300ms",
+  OutAnimationDuration: "1000ms",
+  ReducedAnimationDuration: "300ms",
+
   PenInsetBlockEnd: theme.SpaceScale3,
   PenInsetInlineEnd: theme.SpaceScale3,
   PenRowGap: theme.SpaceScale1,
-  ReducedAnimationDuration: "300ms",
-  RowGap: 0,
-  SuccessBackgroundColor: theme.ColorBackgroundSuccessDark,
-  TextColor: theme.ColorTextBodyInverse,
+
+  CautionBackgroundColor: theme.ColorBackgroundCautionLight,
+  CautionBorderColor: theme.ColorBackgroundCautionBase,
+  CautionIconColor: theme.ColorCautionDark,
+
+  DangerBackgroundColor: theme.ColorBackgroundDangerLight,
+  DangerBorderColor: theme.ColorBackgroundDangerBase,
+  DangerIconColor: theme.ColorDangerBase,
+
+  SuccessBackgroundColor: theme.ColorBackgroundSuccessLight,
+  SuccessBorderColor: theme.ColorBackgroundSuccessBase,
+  SuccessIconColor: theme.ColorSuccessBase,
```

Comment:
hmmm not worth slowing down the PR here but I'm worried about the frequency of thrash for publishing new breaking changes each week to downstream teams we'll want to start finding a safer way to make updates like this! Either publishing codemods, or using js to route the same historical value to something new...